third_party/accessibility/ax/platform/ax_platform_node_mac.mm

```diff
@@ -2,26 +2,24 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
I have to completely rewrite this file because the upstream ax_platform_node_mac.mm  is still WIP is not feature complete